### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Thoth1111**
### Popup Window

In this branch, I refactored the project section so that the projects section is created dynamically using the information stored in an array of JavaScript objects in the myscript.js file. With my learning partner's assistance, we then created a function for each 'See Project' button that extracted details from the projects sections, made some additions and ommissions then displayed them in a pop-up window; with different layouts for the mobile and desktop versions.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -282,7 +392,9 @@
     transition: background-color 0.5s, width 0.5s, color 0.5s;
   }
 
-  .button-1:hover {
+  .button-1:hover,
+  .button-2:hover,
+  .button-3:hover {
     box-shadow: 0 8px 16px;
     border-color: rgba(255, 97, 99, 0.24);
     background-color: aqua;
```
Comment (AmaduKamara) [inline style.css]: - [x] To match the original design, I noticed your hover styles are not the same as the original design styles. Kindly take off the border from the buttons, and adjust the background color for the buttons on hover to match the design and not the aqua color you're having. 

**Your design**

Screenshot 2022-12-23 at 3 07 36 AM

**The original design**

Kindly modify your button styles 

Screenshot 2022-12-23 at 3 14 10 AM

Please use these colors from the template to match your design with the original design given on the hover of a button. In your case, you'll use the green color. 👍🏽

Screenshot 2022-12-23 at 3 12 55 AM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Thoth1111,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used GHitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Popup details rendered dynamically 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (Thoth1111) [reply style.css]: Thank you for the helpful feedback. I have made the requested changes and set the button borders and outlines to none. I've also altered the hover styling of the buttons to match the figma template.

**Review by TadesseDev — APPROVED** [review]
### Hi @Thoth1111 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [x]  Correct GitHub Flow 👍🏾
 - [x]  Good commit history 🥇
 - [x]  Good PR title/description 💯
 - [x]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**